02-Reproject-Raster-In-R.Rmd

```diff
@@ -231,12 +244,11 @@ res(DTM_hill_UTMZ18N_HARV)
 Once we have reprojected the raster, we can try to plot again!
 
 ```{r plot-projected-raster }
-
 #plot newly reprojected hillshade
 plot(DTM_hill_UTMZ18N_HARV,
     col=grey(1:100/100),
     legend=F,
-    main="NEON Hillshade - DTM\n Harvard Forest")
+    main="NEON DTM with Hillshade\n Harvard Forest")
```

Comment:
Let's decide on how we want to title figures and then make changes throughout the lessons (including Vector and Time Series) as a different PR.  Would we like to say the the DTM is NEON's (e.g. start all titles with NEON)  or that the DTM is of the NEON's Harvard Forest Field Site.   
My two cents: 
First line what is the figure of.
Second (last) line: NEON XXX  Field Site  (where XXX is Harvard Forest or SJER)
a middle line is added as needed to clarify 1st line (e.g. when comparisons are made between to plots -- see histograms in Lesson 00).  

There is inconsistency across all lessons and I'd rather fix at one time then piece-meal which might just introduce more inconsistency.